gribscan/rotated_grid.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import division
+
+import numpy as np
+import eccodes as ec
+
+pi = np.pi
+rad = pi / 180.0
+radinv = 1.0 / rad
+
+
+def rot_to_reg(pole_lon, pole_lat, lon, lat):
+    """Rotates from rotated grid to regular grid
+
+    Parameters
+    ----------
+    pole_lon : float
+        Longitude of pole in degrees
+    pole_lat : float
+        Latitude of pole in degrees
+    lon : float, list
+        Longitudes of points in grid. Can also be a single point.
+    lat : float, list
+        Latitudes of points in grid. Can also be a single point.
+
+    Returns
+    -------
+    lon1 : float, list
+        Longitudes of regular grid. Same shape as input.
+    lat1 : float, list
+        Latitudes  of regular grid. Same shape as input.
+    """
+    sin_pole = np.sin(rad * (pole_lat + 90.0))
+    cos_pole = np.cos(rad * (pole_lat + 90.0))
+
+    sin_lon = np.sin(rad * lon)
+    cos_lon = np.cos(rad * lon)
+    sin_lat = np.sin(rad * lat)
+    cos_lat = np.cos(rad * lat)
+
+    lat_tmp = cos_pole * sin_lat + sin_pole * cos_lat * cos_lon
+
+    try:
+        lat_tmp[np.where(lat_tmp < -1.0)] = -1.0
+        lat_tmp[np.where(lat_tmp > 1.0)] = 1.0
+    except TypeError:
+        if lat_tmp < -1.0:
+            lat_tmp = -1.0
+        if lat_tmp > 1.0:
+            lat_tmp = 1.0
```

Comment:
This is maybe more ideomatic (and I'd expect it to take care of the `TypeError`):
```suggestion
    lat_tmp = np.clip(lat_tmp, -1.0, 1.0)
```